catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/fs/FileSystemUtils.java

```diff
@@ -41,12 +41,27 @@ public static Map<String, FileSystemProvider> getFileSystemProviders(String file
     ServiceLoader<FileSystemProvider> allFileSystemProviders =
         ServiceLoader.load(FileSystemProvider.class);
 
+    if (null == fileSystemProviders) {
```

Comment:
Since we already removed this `fileSystemProviders` configuration, why do we still need to pass in this argument?

Reply:
I think we can simplify this method since we load all the fs providers.

Reply:
This method will be also utilized by Gravitino server, in the server side, file system providers are required if we want to access cloud storage

Reply:
I would suggest you to figure out a better way, use `null` value to check and differentiate different code path is not a good way.